projects/dotcms-field-elements/src/components/dot-date/dot-date.tsx

```diff
@@ -10,7 +10,11 @@ import {
     Watch
 } from '@stencil/core';
 import Fragment from 'stencil-fragment';
-import { DotFieldStatusClasses, DotFieldStatusEvent, DotFieldValueEvent } from '../../models';
+import {
+    DotFieldStatusClasses,
+    DotFieldValueEvent,
+    DotInputCalendarStatusEvent
```

Comment:
Shouldn't we move this to a `/models` folder inside `dot-input-calendar`?

Reply:
it will be used in 4 custom fields in total.